java/apirag-bedrock/src/main/java/com/datasqrl/ai/models/bedrock/BedrockChatProvider.java

```diff
@@ -142,4 +143,23 @@ private JSONObject promptBedrock(BedrockRuntimeClient client, String modelId, St
     System.out.println("🤖Bedrock Response:\n" + jsonObject);
     return jsonObject;
   }
+
+  public static ResponseMessage toResponse(BedrockChatMessage msg) {
```

Comment:
I thought ResponseMessage is an ApiModel that is only fit for our SimpleServer Example. I don't think it should be in the core packages.